lib/interactive.dart

```diff
@@ -160,28 +160,23 @@ class Interactive {
     width = col.clientWidth;
     container.remove();
 
-    if (width < MIN_CHART_WIDTH) {
-      width = MIN_CHART_WIDTH;
-    }
-
-    return width;
+    return width < MIN_CHART_WIDTH ? MIN_CHART_WIDTH : width;
   }
 
   String _getThemeColor(String theme) {
-    String color = "black";
-    _themes.forEach((t) {
+    String color = "silver";
+    for (var t in _themes) {
       if (t.value == theme) color = t.color;
-    });
+    }
     return color;
   }
 
   String _lookupFilterLabels(String filter, String option) {
     String label = "Unknown";
     for (var f in _filters) {
-      if (f.value != filter) continue;
-      for (var o in f.options) {
-        if (o.value == option) {
-          return o.label;
+      if (f.value == filter) {
```

Comment:
I would keep here the previous version, ie `if (f.value != filter) continue;` as an early 'exit' from the iteration. Maybe a bit less readable, but a bit more bug resistant.

Reply:
Ahh got it, fixed